src/main/java/up/value/chefleaserver/dto/PopupDetailGetResponse.java

```diff
@@ -0,0 +1,38 @@
+package up.value.chefleaserver.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDate;
+import java.util.List;
+import up.value.chefleaserver.domain.Popup;
+
+public record PopupDetailGetResponse(
+        String popupImage,
+        String popupName,
+        List<String> popupCategories,
+        String popupAddress,
+        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "YY.MM.dd")
+        LocalDate popupPeriod,
+        HeadChefInfoResponse headChefInfo,
+        List<MenuResponse> menus,
+        Boolean isLiked
+) {
+    public static PopupDetailGetResponse of(Popup popup, HeadChefInfoResponse headChefInfo, Boolean isLiked) {
+        return new PopupDetailGetResponse(
+                popup.getImage(),
+                popup.getName(),
+                popup.getPopupCategories().stream()
+                        .map(popupCategory -> popupCategory
+                                .getCategory()
+                                .getKoreanLabel())
+                        .toList(),
```

Comment:
메서드 참조 쓰면 좀 더 가독성 좋게 가능할 것 같아요!
람다식 대신 메서드 참조!

```suggestion
                popup.getPopupCategories()
                        .stream()
                        .map(PopupCategory::getCategory)
                        .map(Category::getKoreanLabel)
                        .toList(),
```